Mathlib/NumberTheory/Transcendental/AnalyticPart.lean

```diff
@@ -0,0 +1,606 @@
+/-
+Copyright (c) 2025 Michail Karatarakis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michail Karatarakis
+-/
+module
+
+public import Mathlib.Analysis.Analytic.Order
+public import Mathlib.Analysis.Complex.CauchyIntegral
+public import Mathlib.Analysis.Normed.Module.Connected
+
+/-!
+Some auxiliary lemmata covering the analytic part of the proof of the Gelfond–Schneider theorem.
+-/
+
+set_option autoImplicit true
+set_option linter.style.multiGoal false
+set_option linter.unusedVariables false
+
+@[expose] public section
+
+open BigOperators Module.Free Fintype Matrix Set Polynomial Finset Complex Differentiable AnalyticAt
+
+lemma exists_mem_finset_min' {γ : Type _} {β : Type _} [LinearOrder γ]
```

Comment:
Why even have this lemma? It's just `Finset.exists_min_image` with a spurious variable `y` inserted.